content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+      1. Actors: This sticky represents a human interaction and should be placed with policies to identify who is responsible of making the decision and performing the task, a policy without an actor is automated
+      1. Action: An action that must be performed due to policy and produces one or more interesting events
+      1. Read Models: Data required to make decisions or perform commands
+      1. System: external/3rd party tools such as; email, excel, notepad, or any required COTS application
+      Note: it does not always need to happen in this order and more grammar may be added, so it is best to leave enough space or be prepared to shift post-its further to the right as you make changes.
+    - Negative/Alternative Outcomes: If a command can result in multiple or alternative events then all events should be placed vertically
+    - Pain Points and Opportunities: Place a sticky to represent any pain points and any opportunities that are identified at the time. Pain points can be around the process itself or the current system or tools supporting the process
+
+1. Narrate the process again; correct inaccuracies and add missing grammar. You can also encourage the discovery of negative events
+   Repeat this until you are satisfied you have enough of the process to identify a thin slice.
+
+1. Review Questions and Pain Points: Try to answer as many questions as possible and rationalize pain points and opportunities
+
+#### Success/Expected Outcomes
+   - Capture information to assist further in the development process
+     - Ubiquitous Language
+     - Domain Events
+     - Policies for Decision Making 
+     - Actors
+     - Read Models
+     - Manual Actions + System Commands
+     - Integrations with tools and apps
+     - Pain Points + Opportunities
+   - Identify one or more thin slices. Try to identify a simplified end-to-end process that can provide the most value. 
+
+## Related Practices:
+### `EventStorming`
+As mentioned Process Modelling evolved from `EventStorming` and is a fundamental part of the complete the tool, as prescribed by Alberto Brandolini. However the storming phase, or `Big Picture` as referred to by Alberto Brandolini, has been more widely adopted with minor variations to the original, such as the use of [EventStorming](/practices/event-storming)  as described in this Developer Center. Therefore Process Modelling naturally offers a way to enrich any `EventStorming` session
+### UML/C4/Boris
+if you are using an Object Oriented based notation to model your architecture, Process Modelling offers an algorithmic view of the problem you are solving, and can be used to complete a number of diagrams as the process is narrated.
+### Boris and SNAP
+The grammar used by Process Modelling offers a lot of information that can be directly used in the completion of [Boris](/practices/boris) and [SNAP](/practices/boris) exercises. 
+
+### Variations:
```

Comment:
This and both variations / variants are not really easy to read and a bit confusing.

Could those be merged? Would it make sense to explain the differences between the three variants a the very beginning of the article and let the readers choose what they want to apply rather than to have to go back and forth?

```suggestion
## Variants
```